contracts/AdminableProxy.sol

```diff
@@ -8,7 +8,7 @@
 
 pragma solidity ^0.4.24;
 
-import "zos-lib/contracts/upgradeability/AdminUpgradeabilityProxy.sol";
+import "./AdminUpgradeabilityProxy.sol";
```

Comment:
why this change? Is it related to "node modules updated to latest versions" ?

Reply:
Yes, it is. I explained already somewhere that there is a wrong import in the latest version of openzeppelin-eth: https://github.com/zeppelinos/zos/issues/461

Reply:
`zos` uses `openzeppelin-solidity 1.10.0` so it is not really a bug, no? Let's keep `openzeppelin-solidity` to `1.*`

Reply:
Yeah, `openzeppelin-solidity` that's what I meant of course
I'm not sure... Looks like from the new version they will start using 2.0. Which also makes sense since `openzeppelin-eth` they recommend to use is 2.0. And also 2.0 has several important changes including security ones. Besides we already modified some of zos contracts for our needs. Why not update this one as well?

Reply:
I would prefer to maintain the least amount of code as possible. Thus, I'm inclined to remove all the "forks" form openzeppelin and just use 1.10 version. However, I'm not an owner for this repo. @eezcjkr what's your opinion?

Reply:
One more reason: zos is made and tested with OZ 1.10.0, so it is the safest to use the same version